ex/auto/writer/odev_build_form/build_form.py

```diff
@@ -99,22 +104,28 @@ def create_form(self, doc: XTextDocument) -> None:
         )
 
         # buttons, all with listeners
-        props = Forms.add_button(doc=BuildForm.doc, name="first", label="<<", x=2, y=63, width=8)
+        col1_x= 2
```

Comment:
I t would be fine to just use `x` here.
After a button or control is added you can redefine `x` or any other var as needed before adding the following control.

Reply:
No, SPR aims to reflect this fundamental part of the form design with the first section having two columns (although first column is just left margin). Happy to take advice on appropriate variable name.